app/backend/src/clj/teet/contract/contracts_queries.clj

```diff
@@ -112,3 +113,14 @@
   (->> (contract-listing-query db user (cu/without-empty-vals search-params))
       (sort-by :meta/created-at)
       reverse))
+
+(defquery :contracts/project-related-contracts
+  {:doc "Return a list of contracts related to the given project"
+   :context {db :db}
+   :args {project-id :thk.project/id}
+   :project-id project-id
+   :authorization {}}
+  (->> (contract-db/project-related-contracts db [:thk.project/id project-id])
+       (sort-by
+         #((:thk.contract/status %) contract-model/contract-status-order))
```

Comment:
would be clearer imo to `(sort-by (comp contract-model/contract-status-order :thk.contract/status) ...)`

And that won't crash if there is no status.

Reply:
Good catch a lot clearer idd